content/hi/docs/tasks/tools/included/verify-kubectl.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "kubectl इंस्टॉल सत्यापित करें"
+description: "kubectl कैसे सत्यापित करें."
+headless: true
+---
+
+kubectl को कुबेरनेट्स क्लस्टर को खोजने और एक्सेस करने के लिए, उसे
+[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) की आवश्यकता होती है,
+जो स्वचालित रूप से तब बनता है जब आप
+[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) का उपयोग करके क्लस्टर बनाते हैं
+या मिनीक्यूब क्लस्टर को सफलतापूर्वक डेप्लॉय करते हैं.
+डिफ़ॉल्ट रूप से, kubectl कॉन्फ़िगरेशन `~/.kube/config` पर स्थित होता है.
+
+जाँच करें कि क्लस्टर स्टेट प्राप्त करके kubectl ठीक से कॉन्फ़िगर किया गया है:
+
+```shell
+kubectl cluster-info
+```
+
+यदि आप एक URL प्रतिक्रिया देखते हैं, तो kubectl आपके क्लस्टर को एक्सेस करने के लिए सही ढंग से कॉन्फ़िगर किया गया है.
```

Comment:
```suggestion
यदि आपको  एक URL प्रतिक्रिया दिखती  हैं, तो kubectl आपके क्लस्टर तक पहुँचने के लिए सही ढंग से कॉन्फ़िगर हुआ  है।
```

Reply:
made some changes here! do share your thoughts, as this looks more appropriate
One tip: while localizing, imagine that you are reading the sentence for 1st time (as a beginner). Think of it as this way & try to figure out what could be the best fit
Overall, the main purpose is to make the reader understand the concepts